src/components/InputSelector/InputSelector.vue

```diff
@@ -124,3 +130,5 @@ export default {
   },
 };
 </script>
+
+<style src="vue2-perfect-scrollbar/dist/vue2-perfect-scrollbar.css"></style>
```

Comment:
I don't think this will work. The instructions say to import it in script `import 'vue2-perfect-scrollbar/dist/vue2-perfect-scrollbar.css'`